### ISSUE TITLE
FSE: Open "List View" by default when in the site editor

### ISSUE THREAD

**Issue by davipontesblog** [Automattic/wp-calypso#61611]
### What

This issues discusses how expanding List View in the site editor by default may be useful to users.

### Why

- It provides additional context to users when navigating the editor canvas when it comes to the parent/child relationship between blocks,
- It makes selecting the right block to edit a lot easier, especially when it comes to parent and child blocks,
- It makes drag and drop easier for blocks,
- It exposes context by showing html tags for blocks,
- Highlighting blocks in the List view when clicking a block in the Editor shows the name of the block, which is otherwise hidden unless the user has the Block Settings bar (on the right) open.

**Comment by Addison-Stavlo** [comment]
I think this could definitely be beneficial.

One consideration though would be screen size. If running on mobile/tablet the screen width may force the list view to overtake the entire screen. In the case of these smaller screens, we should probably avoid opening the list-view by default?

**Comment by Addison-Stavlo** [comment]
Also note, this could be a good issue for discussion in the core Gutenberg repo.

**Comment by Addison-Stavlo** [comment]
cc @jameskoster - Any discussion or possibility of this being a behavior that would make sense in core as well as dotcom?

**Comment by arinoch** [comment]
Giving this a bump for accessibility reasons. See for context: 34428916-hc

**Comment by jameskoster** [comment]
> Any discussion or possibility of this being a behavior that would make sense in core as well as dotcom?

It hasn't come up yet, but I agree it's worth opening an issue to discuss.

**Comment by Addison-Stavlo** [comment]
I think in general this change should be discussed and surfaced in core.

On dotcom our current plans in creating a guided editor tour should serve to expose new users to the list view and its benefits.

I think we may close this in favor of the tour development, but should discuss further in core regarding the open by default behavior.
